cmd/blockchaincmd/create.go

```diff
@@ -358,6 +359,19 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 	if err = promptValidatorManagementType(app, sc); err != nil {
 		return err
 	}
+	if createFlags.validatorManagerController == nil {
```

Comment:
this seems to be done only for PoA. Can we just check is PoA is selected previous to doing this?